### PR TITLE
additional table to recap 'volume acquisition timing'

### PR DESCRIPTION
**Pull request by Remi-Gau**
See issue #85.

I also marked as `code` some fields that I think had been missed after the import from google doc.

### CODE REVIEW

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -265,10 +265,27 @@ combined image rather than an image from each coil.
...
+| option 2          |       \[ ]     |     \[X]    |                     |    \[ ]   |      \[X]    |
+| option 3          |       \[ ]     |             |         \[X]        |    \[ ]   |      \[X]    |
+| sparse sequence 1 |       \[X]     |     \[X]    |         \[ ]        |           |      \[ ]    |
+| sparse sequence 1 |       \[X]     |     \[X]    |         \[ ]        |    \[X]   |      \[ ]    |
+| sparse sequence 1 |       \[X]     |             |         \[ ]        |    \[X]   |      \[ ]    |
+
+**Legend**
+\[X] --> has to be filled
+\[ ] --> has to be left empty
+empty cell --> can be specified but not required
```
Comment (chrisfilo) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: Single new lines are not rendered as new lines in Markdown. Better turn this into a list.

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ by Ben Inglis:
...
+| EffectiveEchoSpacing           | RECOMMENDED. The "effective" sampling interval, specified in seconds, between lines in the phase-encoding direction, defined based on the size of the reconstructed image in the phase direction. It is frequently, but incorrectly, referred to as "dwell time" (see `DwellTime` parameter below for actual dwell time). It is required for unwarping distortions using field maps. Note that beyond just in-plane acceleration, a variety of other manipulations to the phase encoding need to be accounted for properly, including partial fourier, phase oversampling, phase resolution, phase field-of-view and interpolation.<sup>2</sup> This parameter is REQUIRED if corresponding fieldmap data is present.                                                                                                                  |
```
Comment (Remi-Gau) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: DwellTime  changed to `DwellTime`

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ first "effective" echo and the center of the last "effective" echo.
...
+| SliceEncodingDirection | RECOMMENDED. Possible values: `i`, `j`, `k`, `i-`, `j-`, `k-` (the axis of the NIfTI data along which slices were acquired, and the direction in which `SliceTiming` is defined with respect to). `i`, `j`, `k` identifiers correspond to the first, second and third axis of the data in the NIfTI file. A `-` sign indicates that the contents of `SliceTiming` are defined in reverse order - that is, the first entry corresponds to the slice with the largest index, and the final entry corresponds to slice index zero. When present, the axis defined by `SliceEncodingDirection` needs to be consistent with the ‘slice_dim’ field in the NIfTI header. When absent, the entries in `SliceTiming` must be in the order of increasing slice index as defined by the NIfTI header.                                                                                                                                                                                           |
```
Comment (Remi-Gau) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: SliceTiming  changed to `SliceTiming`

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 4]
```diff
@@ -249,7 +249,7 @@ JSON file.
...
-| VolumeTiming   | REQUIRED. The time at which each volume was acquired during the acquisition. It is described using a list of times (in JSON format) referring to the onset of each volume in the BOLD series. The list must have the same length as the BOLD series, and the values must be non-negative and monotonically increasing. This field is mutually exclusive with RepetitionTime and `DelayTime`. If defined, this requires acquisition time (TA) be defined via either SliceTiming or AcquisitionDuration be defined.                                              |
+| VolumeTiming   | REQUIRED. The time at which each volume was acquired during the acquisition. It is described using a list of times (in JSON format) referring to the onset of each volume in the BOLD series. The list must have the same length as the BOLD series, and the values must be non-negative and monotonically increasing. This field is mutually exclusive with `RepetitionTime` and `DelayTime`. If defined, this requires acquisition time (TA) be defined via either `SliceTiming` or `AcquisitionDuration` be defined.                                        |
```
Comment (Remi-Gau) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: RepetitionTime changed to `RepetitionTime`
SliceTiming  changed to `SliceTiming`
AcquisitionDuration  changed to `AcquisitionDuration`

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 5]
```diff
@@ -265,10 +265,27 @@ combined image rather than an image from each coil.
...
+| DelayTime                         | RECOMMENDED. User specified time (in seconds) to delay the acquisition of data for the following volume. If the field is not present it is assumed to be set to zero. Corresponds to Siemens CSA header field `lDelayTimeInTR`. This field is REQUIRED for sparse sequences using the `RepetitionTime` field that do not have the `SliceTiming` field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with `VolumeTiming`.                                                                                                                               |
```
Comment (Remi-Gau) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: SliceTiming changed to `SliceTiming`

**Comment by Remi-Gau** [comment]
Is that what you meant by comments on changes in the definitions?

Also I created a list for the legend but it seems that github markdown automatically renders that as a tickbox. Trying to find a fix for that but nothing so far. Any idea? Will keep looking.

**Comment by chrisfilo** [comment]
Maybe escaping square brackets with `[ ]`?

**Comment by Remi-Gau** [comment]
Found a way to fix it.
Preview is available here. https://github.com/Remi-Gau/bids-specification/blob/table_vol-timing/src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md#other-recommended-metadata
Sorry for the messy history: took a bit of trial and error.

**Comment by chrisfilo** [comment]
I'm looking at the table:
- isn't `sparse sequence 2` and `sparse sequence 3` different just by `SliceTiming` being optional? can't they be combined into one row with `SliceTiming` having an empty cell?
- are options 1-3 allowed for sparse sequences? it's not clear from the naming scheme

**Comment by Remi-Gau** [comment]
>     * isn't `sparse sequence 2` and `sparse sequence 3` different just by `SliceTiming` being optional? can't they be combined into one row with `SliceTiming` having an empty cell?

Hm... I think you are right if I stick to my own legend then I can just remove the sparse sequence 2.

>     * are options 1-3 allowed for sparse sequences? it's not clear from the naming scheme

I think that only option 2 would be actually usable for sparse sequences. Do you agree? I am copying the below the criteria of the definitions. 

I will push something that fixes thoses: also just changing the naming to option A-E and mentioning in the legend what options are OK for sparse sequences.

> SliceTiming is REQUIRED for sparse sequences that do not have the DelayTime field set.
> RepetitionTime  is mutually exclusive with VolumeTiming
> VolumeTiming is mutually exclusive with RepetitionTime and DelayTime. If defined, this requires acquisition time (TA) be defined via either SliceTiming or AcquisitionDuration be defined.
> DelayTime is REQUIRED for sparse sequences using the RepetitionTime field that do not have the SliceTiming field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with VolumeTiming.
> AcquisitionDuration field is REQUIRED for sequences that are described with the VolumeTiming field and that not have the SliceTiming field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with RepetitionTime.

**Pull request merged**